projects/1.18/assets/applied-mekanistics/appmek/lang/en_us.json

```
@@ -0,0 +1,24 @@
+{
+  "itemGroup.appmek": "应用能源：通用机械附属",
+
+  "gui.appmek.chemical": "化学品",
+
+  "item.appmek.chemical_cell_housing": "ME化学品原件外壳",
+  "item.appmek.creative_chemical_cell": "创造型ME化学品储存元件",
+
+  "item.appmek.chemical_storage_cell_1k": "1k-ME化学品储存元件",
+  "item.appmek.chemical_storage_cell_4k": "4k-ME化学品储存元件",
+  "item.appmek.chemical_storage_cell_16k": "16k-ME化学品储存元件",
+  "item.appmek.chemical_storage_cell_64k": "64k-ME化学品储存元件",
+  "item.appmek.chemical_storage_cell_256k": "256k-ME化学品储存元件",
+
+  "item.appmek.portable_chemical_storage_cell_16k": "16k便携化学品元件",
+  "item.appmek.portable_chemical_storage_cell_1k": "1k便携化学品元件",
+  "item.appmek.portable_chemical_storage_cell_4k": "4k便携化学品元件",
+  "item.appmek.portable_chemical_storage_cell_64k": "64k便携化学品元件",
+  "item.appmek.portable_chemical_storage_cell_256k": "256k便携化学品元件",
+
+  "item.appmek.chemical_p2p_tunnel": "P2P通道-化学品",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "item.appmek.chemical_p2p_tunnel": "P2P通道 - 化学品",
```